pages/index.js

```diff
@@ -60,4 +30,46 @@ const Button = styled.button`
   border-radius: 3px;
   margin: 2px;
   padding: 0.5px;
+  display: center;
+  text-align: center;
+`;
+
+const StyledLink = styled.p`
+  display: center;
+  text-align: center;
+  font-size: 2em;
+  color: blue;
```

Comment:
You wrote this component twice. This is the same case where you should extend the next Link component. And if you use it twice, put it in a seperate file and import it where you need it from there.